### DISCUSSION TITLE
Docs: explain the use of `extend` in configuration

### DISCUSSION THREAD

**Discussion by RDeluxe** [chakra-ui/panda#1111] [General]
### Subject

Presets

### Description

After struggling a while to understand why my preset was completely overriding the base one (even after installing panda base preset then adding it to the array) I realised that I had forgot the keyword `extend` in my config.

I don't think this is explained anywhere, but as far as I understand without this keyword there is not smart merging. I think it deserves more explanation in the doc.

Thanks!

**Comment by astahmer** [comment]
good point
also, would it help having a warning message if NOT using the extend keyword ?

**Comment by RDeluxe** [comment]
Not sure. I think it could be a valid pattern in some case, but not in most, that's true. 

But take my pov with a grain of salt, as I'm not 100% sure I understand all the moving parts in Panda yet.
